presto-docs/src/main/sphinx/release/release-0.221.rst

```diff
@@ -0,0 +1,46 @@
+=============
+Release 0.221
+=============
+
+General Changes
+---------------
+* Add a variant of :func:`strpos` to return the position of the N-th instance of the substring.
+* Add :func:`strrpos` that returns position of the N-th instance of substring from the back of a string.
+* Add aggregation function :func:`entropy`.
+* Add classification aggregation functions :func:`classification_miss_rate`, :func:`classification_precision`,
```

Comment:
Same as above?

Reply:
@tdcmeehan can you give a brief description?

Reply:
I think it would be hard to provide brief one-line explanation for these.  I think it's a better idea to just directly link to the documentation that was added for them.  See for example ST_Points from 0.217: https://prestodb.github.io/docs/current/release/release-0.217.html